frontend/src/components/where-to-buy/where-buy-item/where-buy-item.tsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect, useRef } from 'react';
+
+import logo from '@assets/images/logo-autoria.png';
+import mapPin from '@assets/images/map-pin.svg';
+import speedometr from '@assets/images/speedometr.svg';
+import { AutoRiaLinks } from '@common/enums/app/app';
+import { WhereBuyInterface } from '@common/types/where-to-buy/where-to-buy';
+import { ButtonOutline } from '@components/common/button-outline/button-outline';
+import { formatPrice } from '@helpers/helpers';
+import { ExpandLess, ExpandMore } from '@mui/icons-material';
+import Divider from '@mui/material/Divider';
+import { clsx } from 'clsx';
+
+import styles from './styles.module.scss';
+
+interface WhereBuyItemProps {
+  poster: WhereBuyInterface;
+}
+
+const WhereBuyItem: React.FC<WhereBuyItemProps> = (props) => {
+  const [open, setOpen] = useState(false);
+  const [height, setHeight] = useState(0);
+  const elementRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    const { current: appElement } = elementRef;
+    setHeight(appElement?.clientHeight as number);
+  }, []);
+
+  const {
+    USD: price,
+    autoData: { description, year, raceInt: race },
+    linkToView,
+    title,
+    color: { hex: color },
+    stateData: { regionNameEng: location },
+  } = props.poster;
+  const url = `${AutoRiaLinks.LINK_ADVERT}${linkToView}`;
+  const formatedPrice = formatPrice(price);
+
+  const handleBuy = (): void => {
+    window.open(url, '_blank');
+  };
+
+  return (
+    <div>
+      <div className={styles.container}>
+        <div className={styles.logo}>
+          <img src={logo} alt="logo" />
+        </div>
+        <div className={styles.description}>
+          <span className={styles.carName}>{title}</span>
+          <div className={styles.labelGroup}>
+            <div className={styles.label}>
+              <img src={speedometr} alt="speedometr" />
+              {`${race} 000 km`}
```

Comment:
Are you sure that race always would be in km not miles?

Reply:
Yes, it is according autoria documantation.